File: Lua/ascii.lua
function numeroParaASCII(numero)
    if numero < 0 or numero > 127 then
        return "Valor invalido!"
    end
    return string.char(numero)
end

io.write("Digite um numero (0-127): ")
local numero = tonumber(io.read())
print("ASCII: " .. numeroParaASCII(numero))

File: Dart/ascii.dart
import 'dart:io';

String numeroParaASCII(int numero) {
  if (numero < 0 || numero > 127) return 'Valor invalido!';
  return String.fromCharCode(numero);
}

void main() {
  stdout.write('Digite um numero (0-127): ');
  int numero = int.parse(stdin.readLineSync()!);
  print('ASCII: ${numeroParaASCII(numero)}');
}
